src/System.Numerics.Vectors/ref/System.Numerics.Vectors.netcoreapp.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+
+namespace System.Numerics
+{
+    public partial struct Vector<T> : System.IEquatable<System.Numerics.Vector<T>>, System.IFormattable where T : struct
+    {
+        public Vector(Span<T> values) { throw null; }
```

Comment:
Nit: I do not think we split the `ref` assemblies in multiple files. We tend tend to prefer ifdefs for these.

Reply:
Done for the ref project...

Reply:
I've been given the opposite feedback in the past.  We split the `ref` assembly .cs file here:

https://github.com/dotnet/corefx/blob/master/src/System.Net.Sockets/ref/System.Net.Sockets.netcoreapp.cs

Reply:
A counter example: ttps://github.com/dotnet/corefx/blob/master/src/System.Security.Cryptography.OpenSsl/ref/System.Security.Cryptography.OpenSsl.cs#L23

Reply:
I was asked to put them in a separate file here:  https://github.com/dotnet/corefx/pull/25388#discussion_r152152303

My (weak) opinion is that we should be consistent throughout our code. In our implementation code we shy away from #if and opt for separate files, so I think we should follow the same pattern in our ref code.

Reply:
> Is it a case that the facade definitions are intended to be kept in single file? 

Yes, the reference assembly sources are intentionally kept in single file. They do not follow the conventions used for implementation.

> In our implementation code we shy away from #if and opt for separate files

This make sense and works great for Windows/Unix distinction. This distinction is forward looking, it is actively maintained.

It works not so well for the `.netcoreapp.cs`. The split of netcoreapp is backward looking. We frequently split netcoreapp initially and than abandon the non-netcoreapp flavor. We have many `.netcoreapp.cs` files in the repo that do make sense anymore because of the whole project is netcoreapp-specific, and there is netcoreapp-specific stuff in the main .cs as well.  Hard to understand when you see one of these for the first time.